magit-todos.el

```diff
@@ -137,13 +137,8 @@ details about how section maps work.")
 
 (defvar magit-todos-follow-map
   (let ((map (make-sparse-keymap)))
-    (define-key map (kbd "n") 'magit-todos-next-todo)
-    (define-key map (kbd "C-n") 'magit-todos-next-todo)
-    (define-key map (kbd "<down>") 'magit-todos-next-todo)
-
-    (define-key map (kbd "p") 'magit-todos-previous-todo)
-    (define-key map (kbd "C-p") 'magit-todos-previous-todo)
-    (define-key map (kbd "<up>") 'magit-todos-previous-todo)
+    (define-key map [remap magit-section-forward] #'magit-todos-next-todo)
```

Comment:
I'm not sure that `magit-section-forward`/`-backward` are the right commands to remap.  I think that will break their use outside of `magit-todos` sections.  Can you confirm this?

Instead of having a `magit-todos-follow-map`, we might need to bind keys in the section-specific keymap when the mode is activated.  This may be tricky to get right.

Reply:
I tested it. Outside of `magit-todos` sections, `magit-todos-next-todo` just ignores those invalid items silently, and `magit-section-forward/-backward` are reasonable there. `magit-next-line` and `magit-previous-line` are also OK there, users can bind these extra commands if they want.